com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -137,51 +137,49 @@ void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
             rect.height = EditorGUIUtility.singleLineHeight;
             rect.y += 1;
 
-            var element = m_SerializedCamera.cameras.GetArrayElementAtIndex(index);
-
-            var cam = element.objectReferenceValue as Camera;
```

Comment:
Maybe keep this var so you don't have to do this overlayCamera.camer all the time